examples/whylogs_data_profiling/README.md

```diff
@@ -33,109 +33,131 @@ engineering, data processing, model training and validation or inference. What
 is important is how ZenML allows you to effortlessly add whylogs profiling
 capabilities to all the points in your ML pipeline where data is involved.
 
-The first step in the pipeline shows how applying the `enable_whylogs`
-decorator to an existing step adds the `whylogs` data profiling extension
-to the step context. The whylogs data profiles are returned as step artifacts
-which will be versioned and persisted in the Artifact Store just as any other
-artifacts.
+The first step in the pipeline shows how whylogs data profiles can be generated
+and returned as step artifacts which will be versioned and persisted in the
+Artifact Store just as any other artifacts.
+
+It also shows how the `enable_whylabs` decorator can be applied to an existing
+to automatically log all returned whylogs data profiles to the Whylabs platform.
+This needs to be combined with configuring secrets in the whylogs Data Validator
+stack component to work, as detailed in the [Run it locally](#🖥-run-it-locally)
```

Comment:
does this emoji-in-a-link work? cool, if so...

Reply:
It did work in Visual Studio, but not in GitHub I see. I'll fix it.